### ISSUE TITLE
Exception when encountering blocks inside a mixin call

### ISSUE THREAD

**Issue by evalica** [SomMeri/less4j#282]
Not sure if this issue's title is correct so please adjust it as you like.

So after using LESS4J, I've got:
```
Less4jException: Could not compile less. 4 error(s) occurred:
ERROR 3:22 no viable alternative at input '(' in abstract mixin or namespace (which started at 2:1)
 2: .variations(@extra, @property, @default) {
 3:   .generic-variations(@extra, @default, {
 4:       @{property}: @material-color;

ERROR 3:41 mismatched input '{' expecting '@<variable name>' in abstract mixin or namespace (which started at 3:3)
 2: .variations(@extra, @property, @default) {
 3:   .generic-variations(@extra, @default, {
 4:       @{property}: @material-color;

...
```

for this kind of code:
```
// usage: .variations(~" .check", color, transparent);
.variations(@extra, @property, @default) {
  .generic-variations(@extra, @default, {
      @{property}: @material-color;
    });
}
```
The whole LESS can be found at https://github.com/FezVrasta/bootstrap-material-design/blob/master/less/_mixins-fullpalette.less

For me is not a serious / critical issue since I was just experimenting, but maybe is something you want to get fix. 

Thanks for your work.

**Comment by SomMeri** [comment]
Thank you for the report, I will do bugfixing soon.

**Comment by evalica** [comment]
Thank you :)

**Comment by SomMeri** [comment]
Just released 1.12.0 version should have this fixed. If it is not, please re-open the issue.
